src/Sarif/Microsoft.Json.Schema.ToDotNet.targets

```diff
@@ -16,6 +16,7 @@
   <!-- This target runs only when we actually need to build the object model files, which should be
        only when the JSON schema changes. -->
   <Target Name="BuildObjectModel" Inputs="@(JsonSchemaFile)" Outputs="$(_ObjectModelOutputDirectory)\*.cs">
+    <RemoveDir Directories="$(_ObjectModelOutputDirectory)" />
```

Comment:
Just moving the "Delete" task here wasn't enough. I had to remove the entire directory to get it to work. Don't know why.